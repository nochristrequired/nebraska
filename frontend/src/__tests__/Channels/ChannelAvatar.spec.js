import { render } from '@testing-library/react';
import React from 'react';
import ChannelAvatar from '../../components/Channels/ChannelAvatar';

describe('Channel Avatar', () => {
  it('should render avatar with correct color', () => {
    const { asFragment } = render(<ChannelAvatar color="#fff">ABC</ChannelAvatar>);
    expect(asFragment()).toMatchSnapshot();
  });
});
