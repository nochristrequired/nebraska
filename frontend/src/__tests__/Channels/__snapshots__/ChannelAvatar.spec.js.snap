// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Channel Avatar should render avatar with correct color 1`] = `
<DocumentFragment>
  <div
    class="MuiAvatar-root MuiAvatar-circle makeStyles-colorAvatar-1 makeStyles-colorAvatar-2 MuiAvatar-colorDefault"
  >
    ABC
  </div>
</DocumentFragment>
`;
